opentech/static_src/src/app/src/components/NotesPanel.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import LoadingPanel from '@components/LoadingPanel';
+
+export default class NotesPanel extends React.Component {
```

Comment:
This shares a lot of functionality with the listing component. 

I think i would like to try and make the components less specific in that instance. 

So the NoteListing container would be:
```javascript

renderNote(id) {
    // assuming Note is connected to the store with a selector on ID
    // see https://github.com/reduxjs/reselect#accessing-react-props-in-selectors
    return <Note id=id>
}

error = <>
                <p><strong>Something went wrong!</strong>Sorry we couldn&rsquo;t load notes</p>
                <a onClick={handleRetry}>Refresh</a>.
            </>

<Listing 
    items={[<array of ids>]} // From `fetchNotesForSubmission`
    renderItem={renderNote}
    isError={isErrored}
    errorItem={error}
>

// ----------
<NoteListing submissionID=submissionID>
```